include/gmsh.tcc

```diff
@@ -110,14 +115,14 @@ void GMSH::read_elements(const std::string& filename) {
 
         //! \brief Check element type
         //! \details If element type not == to Tdim, skip element
-        if (elementtype != ndimension) {
-          istream >> line;
-        } else {
-          istream >> elementarray.at(0) >> elementarray.at(1) >>
-              elementarray.at(2) >> elementarray.at(3);
-          elements_.emplace_back(
-              new Point<ndimension>(elementid, elementarray));
-        }
+        // if (elementtype != ndimension) {
```

Comment:
I need to generate the stress to construct `point`. Let me just comment it out first then I will come back to the `elements`